app/src/vanilla/java/cm/aptoide/pt/app/DetailedAppViewModel.java

```diff
@@ -0,0 +1,335 @@
+package cm.aptoide.pt.app;
+
+import cm.aptoide.pt.dataprovider.model.v7.GetAppMeta;
+import cm.aptoide.pt.dataprovider.model.v7.Malware;
+import cm.aptoide.pt.dataprovider.model.v7.Obb;
+import cm.aptoide.pt.dataprovider.model.v7.store.Store;
+import cm.aptoide.pt.view.app.AppDeveloper;
+import cm.aptoide.pt.view.app.AppFlags;
+import cm.aptoide.pt.view.app.DetailedApp;
+import cm.aptoide.pt.view.app.DetailedAppRequestResult;
+import java.util.List;
+
+/**
+ * Created by D01 on 07/05/18.
+ */
+
+public class DetailedAppViewModel {
+
+  private final GetAppMeta.Media media;
+  private final String modified;
+  private final String appAdded;
+  private final Obb obb;
+  private final GetAppMeta.Pay pay;
+  private final String wUrls;
+  private final boolean isPaid;
+  private final String uName;
+  private final boolean isStoreFollowed;
+  private final DetailedApp detailedApp;
+  private final long appId;
+  private final String appName;
+  private final Store store;
```

Comment:
We shouldn't use this, this adds the dataprovider dependency to this class.

This could probably be a flatten in the viewmodel itself